episodes/07-thresholding.md

```diff
@@ -348,13 +348,17 @@ plt.xlim(0, 1.0)
 ![](fig/maize-root-cluster-histogram.png){alt='Grayscale histogram of the maize root image'}
 
 The histogram has a significant peak around 0.2, and a second,
```

Comment:
```suggestion
The histogram has a significant peak around 0.2 and then a broader "hill" around 0.6 followed by a smaller peak near 1.0. Looking at the grayscale image, we can identify the lower peak at 0.2 as the background and the larger pixel values as the foreground, but it is not so obvious what the threshold should be.
```

Reply:
I recommend removing "lower" from your explanation above ("the lower peak at 0.2), because "lower" could have two different meanings in this context. I think "the peak at 0.2" is fine.

Reply:
Yes, "lower peak" is confusing here. I would further suggest:
```diff
-we can identify the lower peak at 0.2 as the background
+we can identify the peak at 0.2 with the background
```
and
```diff
-the larger pixel values as the foreground
+the larger pixel values with the foreground
```
because we're associating pixel values with image features (rather than saying they're identical).